<?xml version="1.0"?>
<launch>
	<!-- Node: Local obstacles builder -->
	<node pkg="mrpt_local_obstacles" type="mrpt_local_obstacles_node" name="mrpt_local_obstacles_node" output="screen">
		<param name="source_topics_2dscan" value="laser1,laser2"/>
                <param name="source_topics_depthcam" value="camera1/depth/points"/>
        </node>
	<!-- Node: Pure Reactive Navigator -->
	<node pkg="mrpt_reactivenav" type="mrpt_reactivenav_node" name="mrpt_reactivenav_node" output="screen">
		<!-- Set type of reactive navigator (2D or 3D) -->
		<param name="nav_type" value="3D"/> 

		<!-- *Important*: This external config file holds the most important navigation settings -->
		<param name="cfg_file_reactive" value="$(find mrpt_reactivenav)/tutorial/reactive3d_config.ini"/>
		<!-- param name="cfg_file_reactive" value="$(find mrpt_reactivenav)/tutorial/reactive3d_config.ini"/ -->  

		<!-- This is to allow the reactive responsive to RVIZ GUI commands -->
		<remap from="reactive_nav_goal" to="/move_base_simple/goal" />
                <remap from="/cmd_vel" to="/giraff_controller/cmd_vel" />

		<!-- Enable logs to ~/.ros/reactivenav.logs , use navlog-viewer to open them -->
<!--		<param name="save_nav_log" value="true"/> -->

		<!-- Robot shape: Load from geometry_msgs::Polygon topic >
		<param name="topic_robot_shape" value="/chassis_polygon" /-->
	</node>
</launch>